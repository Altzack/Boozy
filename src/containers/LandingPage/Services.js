import React, { Component } from 'react';
import styled from 'styled-components/macro';
import {
  FaGrinBeam,
  FaHourglassHalf,
  FaLightbulb,
  FaWineGlassAlt,
} from 'react-icons/fa';

const Title = styled.h1`
  font-size: 2rem;
  letter-spacing: 3px;
  text-transform: capitalize;
  color: rgba(232, 230, 227, 0.85);
  margin-bottom: 30px;
`;
export default class Services extends Component {
  state = {
    services: [
      {
        icon: <FaLightbulb />,
        title: 'Unlimited possibilites',
        info:
          'Put your creativity to work and come up with completely unique drink recipes.',
      },
      {
        icon: <FaHourglassHalf />,
        title: 'Quick and easy',
        info:
          'Looking for something new to try? Boozy can give you great suggestions, fast.',
      },
      {
        icon: <FaWineGlassAlt />,
        title: 'Find your new favorite',
        info:
          'Browse other users awesome creations and find your new favorite drink!',
      },
      {
        icon: <FaGrinBeam />,
        title: 'Sharing is caring',
        info: 'Best of all, you can share your ideas with people everywhere!',
      },
    ],
  };
  render() {
    return (
      <section className="services">
        <Title>Why Boozy?</Title>
        <div className="services-center">
          {this.state.services.map((item, index) => {
            return (
              <article key={index} className="service">
                <span>{item.icon}</span>
                <h6>{item.title}</h6>
                <p>{item.info}</p>
              </article>
            );
          })}
        </div>
      </section>
    );
  }
}
