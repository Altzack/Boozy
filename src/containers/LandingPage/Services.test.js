import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Services from './Services';
import setupTests from '../../setupTests';

it('renders a Serivces by default', () => {
  const wrapper = shallow(<Services />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
