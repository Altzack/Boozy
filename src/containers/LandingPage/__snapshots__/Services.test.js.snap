// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Serivces by default 1`] = `
<section
  className="services"
>
  <Services__Title>
    Why Boozy?
  </Services__Title>
  <div
    className="services-center"
  >
    <article
      className="service"
      key="0"
    >
      <span>
        <FaLightbulb />
      </span>
      <h6>
        Unlimited possibilites
      </h6>
      <p>
        Put your creativity to work and come up with completely unique drink recipes.
      </p>
    </article>
    <article
      className="service"
      key="1"
    >
      <span>
        <FaHourglassHalf />
      </span>
      <h6>
        Quick and easy
      </h6>
      <p>
        Looking for something new to try? Boozy can give you great suggestions, fast.
      </p>
    </article>
    <article
      className="service"
      key="2"
    >
      <span>
        <FaWineGlassAlt />
      </span>
      <h6>
        Find your new favorite
      </h6>
      <p>
        Browse other users awesome creations and find your new favorite drink!
      </p>
    </article>
    <article
      className="service"
      key="3"
    >
      <span>
        <FaGrinBeam />
      </span>
      <h6>
        Sharing is caring
      </h6>
      <p>
        Best of all, you can share your ideas with people everywhere!
      </p>
    </article>
  </div>
</section>
`;
